plumbing/api.go

```diff
@@ -145,6 +153,16 @@ func (api *API) MessageWait(ctx context.Context, msgCid cid.Cid, cb func(*types.
 	return api.msgWaiter.Wait(ctx, msgCid, cb)
 }
 
+// PubSubSubscribe subscribes to a libp2p topic
```

Comment:
i think it says libp2p topic to indicate to the reader that this is a network pubsub, not something local to this process. i think that is a good idea, though i might say 'subscribes to a topic for notifications from the filecoin network'